docs/DATA_FORMATS_INPUT.md

```diff
@@ -174,6 +207,19 @@ For example, if the following configuration:
     "my_tag_1",
     "my_tag_2"
   ]
+
+  ## List of field names to extract from JSON and add as string fields
+  # string_fields = []
+
+  ## gjson query path to specify a specific chunk of JSON to be parsed with 
+  ## the above configuration. If not specified, the whole file will be parsed
+  # json_query = ""
+
+  ## holds the name of the tag of timestamp
+  json_time_key = "b_time"
+
+  ## holds the format of timestamp to be parsed
+  json_time_format = "02 Jan 06 15:04 MST"
```

Comment:
This is an interesting example time format because there it doesn't include a year.  This results in a pared time that is in year 0.  I think after parsing the time we should do something similar to what we did in logparser and use the current year.  While this would be pretty annoying if you actually want to store in a metric from year 0, I think it is much more likely to be what is wanted:
```
if ts.Year() == 0 {
	ts = ts.AddDate(timestamp.Year(), 0, 0)
}
```